readme.md

```diff
@@ -1,32 +1,31 @@
 # Antistasi Extender Example
-This is an example of how you could extend or overwrite antistasi with new maps, templates
+This is an example of how you could extend antistasi with new maps or templates or overwrite existing ones.
 
 # Requirements
   - Arma 3
-  - Arma 3 tools  
+  - Arma 3 Tools  
     
-  Both needs to be run once from steam
+  Both need to be run once from steam.
 
-# How to get started
-- run the `Install.ps1` script and follow the instructions on screen
+# How to get started:
+- Run the `Install.ps1` script and follow the instructions on screen.
 - Run the `Build.ps1` script this will create a folder in the root folder called build and compile your extension into it.
```

Comment:
Missing a full stop in the middle. Could also make a note that the build script needs to be run again each time you change anything.

Reply:
done